### PR TITLE
Tip 6 Tweaks

### PR DESCRIPTION
**Pull request by rasbt**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

Minor additions (loss function as hyperparam choice) and some typo fixes.

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]

File: content/08.hyperparameters.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ## Tip 6: Tune your hyperparameters extensively and systematically {#hyperparameters}
 
-Given at least one hidden layer, a non-linear activation function, and a large number of hidden units [@tag:hornik-approximation], multi-layer neural networks can approximate arbitrary continuous functions that relate input and output variables. 
-Deeper architectures that feature additonal hidden layers and an increasing number of overall hidden units and learnable weight parameters (the so-called increasing "capacity" of neural networks) allow for solving increaingly complex problems.
-However, this increased capacity results in many more parameters to tune, which can pose additional challenges during model training.
+Given at least one hidden layer, a non-linear activation function, and a large number of hidden units, multi-layer neural networks can approximate arbitrary continuous functions that relate input and output variables [@tag:hornik-approximation ; @doi:10.1016/S0893-6080(05)80131-5].
```
Comment (cgreene) [inline content/08.hyperparameters.md:3]:
```suggestion
Given at least one hidden layer, a non-linear activation function, and a large number of hidden units, multi-layer neural networks can approximate arbitrary continuous functions that relate input and output variables [@tag:hornik-approximation; @doi:10.1016/S0893-6080(05)80131-5]. 
```

**Review by cgreene — APPROVED** [review]
🚀

File: content/08.hyperparameters.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+Similar to the Continental Breakfast Included effect discussed in [Tip 2](#baselines), a lack of clarity on how extensive arrays of hyperparameters were tested and/or chosen can affect method developers as they attempt to compare techniques.
 This effect also has implications for those seeking to use existing deep learning methods, as performance estimates from deep neural networks are often provided after tuning.
-The implication of this effect on users of deep neural networks is that attaining performance numbers that match those reported in publications is likely to require signficant effort towards temporally expensive hyperparameter optimization.
+The implication of this effect on users of deep neural networks is that attaining performance numbers that match those reported in publications is likely to require significant effort towards temporally expensive hyperparameter optimization.
 
 Ultimately, to get the best performance of your model, be sure to systematically optimize your hyperparameters on your training dataset, as introduced in the next section.
```
Comment (mdkessler) [inline content/08.hyperparameters.md:16]:
```suggestion
Ultimately, to get the best performance from your model, be sure to systematically optimize your hyperparameters on your training dataset, as introduced in the next section.
```

**Review by mdkessler — APPROVED** [review]
This is really great - awesome edits. Suggested one tiny word change.

File: content/08.hyperparameters.md (hunk 2 above)
Comment (rasbt) [reply content/08.hyperparameters.md:16]: oh yeah, thanks
